README.md

```diff
@@ -65,107 +71,128 @@ and send payments over ILP.
 **Note:** Xpring SDK only works with the X-Address format. For more information about this format, see the [Utilities section](#utilities) and <http://xrpaddress.info>.
 
 ### Wallets
+
 A wallet is a fundamental model object in XpringKit which provides key management, address derivation, and signing functionality. Wallets can be derived from either a seed or a mnemonic and derivation path. You can also choose to generate a new random HD wallet.
 
 #### Wallet Derivation
+
 Xpring-JS can derive a wallet from a seed or it can derive a hierarchical deterministic wallet (HDWallet) from a mnemonic and derivation path.
 
 ##### Hierarchical Deterministic Wallets
+
 A hierarchical deterministic wallet is created using a mnemonic and a derivation path. Simply pass the mnemonic and derivation path to the wallet generation function. Note that you can omit the derivation path and have a default path be used instead.
 
 ```javascript
-const { Wallet } = require("xpring-js");
+const { Wallet } = require('xpring-js')
 
-const mnemonic = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about";
+const mnemonic =
```

Comment:
Did you run this through Prettier or something?

Reply:
I think it just autosaved in VSCode and got formatted nicely.